src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupDetailsPage.jsx

```diff
@@ -123,29 +132,24 @@ export default function UserGroupDetailsPage({ id }) {
   const DeleteGroup = () => (
     <div>
       <WarningAlert message={i18n("UserGroupDetailsPage.delete-warning")} />
-      <form
-        style={{ marginTop: SPACE_SM }}
-        ref={deleteRef}
-        action={setBaseUrl(`/groups/${id}/delete`)}
-        method="POST"
-      >
+      <div style={{ marginTop: SPACE_SM }}>
         <Popconfirm
-          onConfirm={() => deleteRef.current.submit()}
+          onConfirm={deleteUserGroup}
           title={i18n("UserGroupDetailsPage.delete-confirm")}
           okButtonProps={{ className: "t-delete-confirm-btn" }}
         >
           <Button className="t-delete-group-btn" type="primary" danger>
             {i18n("UserGroupDetailsPage.delete-button")}
           </Button>
         </Popconfirm>
-      </form>
+      </div>
     </div>
   );
 
   return (
     <PageWrapper
       title={"User Groups"}
-      onBack={() => navigate(setBaseUrl("/groups"), { replace: true })}
+      onBack={() => navigate(`${baseUrl}`, { replace: true })}
```

Comment:
You don't need to interpolate the baseUrl here just `navigate(baseUrl, {replace:true})`.

Reply:
Same here for JSDoc